public/js/view.js

```diff
@@ -94,78 +94,122 @@ function makeSVG(w, h) {
+ *      (i.e. lines and previously placed stones)
+ * 3. Draw lines
+ * 4. Draw stones
  * 
  * @param {object} board - A GameSpace object
  */
 function draw(board) {
 
-    // Extracting the array representation of the board
+    // 1. Extracting the board representation
     board = board.board.board;
-    // console.log(board);
-
-    // The canvas element
-    var canvas = $("#canvas");
-    var W = 600, H = 600;
-    canvas.css("height", H);
-    canvas.css("width", W);
 
-    // Creating the svg element
-    var svg = $(makeSVG(W, H));
+    // 2. Clearing SVG
+    svg.html("");
 
-    // Draw related variables
-    var scale = W / board.length;
-    var radius = (scale / 2) - 1;
-    var offset = scale / 2;
-
-    // Lines
+    // 3. Drawing lines
     for (var i = 0; i < board.length; i++) {
-
         var line_v = makeLine((scale * i) + offset, offset, (scale * i) + offset, H - offset, "black", "black");
         var line_h = makeLine(offset, (i * scale) + offset, W - offset, (i * scale) + offset, "black", "black");
         svg.append(line_v);
         svg.append(line_h);
     }
 
-    // Stones
+    // 4. Drawing stones
     for (var col in board) {
         for (var row in board[col]) {
-
             if (board[col][row] === 1) {
-                // black stones
+                // black stone
                 var circ = makeCircle((row * scale) + offset, (col * scale) + offset, radius, "black", "black");
                 svg.append(circ);
 
             } else if (board[col][row] === 2) {
-                // white stones
+                // white stone
                 var circ = makeCircle((row * scale) + offset, (col * scale) + offset, radius, "white", "black");
                 svg.append(circ);
             }
-
         }
     }
-
-    // append the svg object to the canvas object.
-    canvas.append(svg);
-
 }
+/******************************* End of drawing *******************************/
+
 
 
 
 
-// Testing
+/********************************** Testing ***********************************/
+// The board
 var myBoard = new GameSpace(9);
+var player = 1;
 
-myBoard.placeToken(1, 2, 5);
-myBoard.placeToken(2, 4, 5);
-myBoard.placeToken(1, 3, 4);
-myBoard.placeToken(2, 3, 6);
-myBoard.placeToken(1, 4, 3);
-myBoard.placeToken(2, 2, 3);
+// Initializing view variables for the given board
+init(myBoard);
 
+// Drawing the board
 draw(myBoard);
 
+// onClick event on the board
+$("#canvas").click(function (e) {
+
+    // Calculating the board coordinates from clicked coordinates
+    var x = Math.floor((e.pageX - $(this).offset().left) / (offset * 2));
+    var y = Math.floor((e.pageY - $(this).offset().top) / (offset * 2));
+
+    // console.log(posX + " " + posY);
+
+    // Placing the token on the board
+    myBoard.placeToken(player, x, y);
+
+    // Draw board
+    draw(myBoard);
+
+    // Swapping players
+    if (player === 1) {
+        player = 2;
+    } else if (player === 2) {
+        player = 1;
+    }
+
+});
+
+
+
+
```

Comment:
I would suggest wrapping the view code in a `View `class.  It's attributes can include the dimensions, and a jQuery reference to the drawing area.

SVGfactory could go in a separate class and file as well. It's good to group related functionality and keep the number of global variables low.